app/helpers/host_initiator_helper/textual_summary.rb

```diff
@@ -0,0 +1,51 @@
+module HostInitiatorHelper::TextualSummary
+  include TextualMixins::TextualGroupTags
+
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    TextualGroup.new(
+      _("Properties"),
+      %i[
+        name
+        ems
+        physical_storage
+      ]
+    )
+  end
+
+  def textual_group_san_addresses
+    san_addresses_values = []
+
+    san_addresses = SanAddress.where(owner_id: @record.id)
```

Comment:
`@record.san_addresses.each` is preferred here, I think.

Reply:
agree. done